pkg/daemon/daemon.go

```diff
@@ -207,40 +199,39 @@ func (d *daemon) AddPeriodicUpdate() {
 				// User allocated guid manually
 				if err = d.guidPool.AllocateGUID(pod.UID, networkId, allocatedGuid); err != nil {
 					failedPods = append(failedPods, pod)
-					glog.Errorf("AddPeriodicUpdate(): %v", err)
+					log.Err(err)
```

Comment:
provide more information. ("failed  to allocate GUID for pod ID error:, %s , %v)